.tool-versions

```diff
@@ -0,0 +1 @@
+yarn 1.22.10
```

Comment:
What does this do?

Reply:
It's help pinning the version of yarn with versioning managers like `mise`or `asdf`